### PR TITLE
Modal message and navigate

### PR DESCRIPTION
**Pull request by etuzimek**

### CODE REVIEW

File: src/components/Cart/Cart.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+  };
+
+  return (
+    <>
+      <div className="cart">
+        <NavLink to="#">
+          <img className="cart__icon" src={LeftIcon} alt="left" />
+        </NavLink>
+        <a className="cart__back">Back</a>
```
Comment (weswojciech) [inline src/components/Cart/Cart.tsx]: Wrap not only a button but also a text since I personally and I think most people will click on a "Back"

Comment (weswojciech) [inline src/components/Cart/Cart.tsx]: Not sure if proper font is used

File: src/components/Cart/Cart.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      setCartItems([]);
+      navigate('/phones');
+    }
+  };
+
+  return (
+    <>
+      <div className="cart">
+        <NavLink to="#">
+          <img className="cart__icon" src={LeftIcon} alt="left" />
```
Comment (weswojciech) [inline src/components/Cart/Cart.tsx:25]: In mockup button is black instead of grey

**Review by weswojciech — CHANGES_REQUESTED** [review]

File: src/components/Cart/Cart.tsx (hunk 1 above)
Comment (etuzimek) [reply src/components/Cart/Cart.tsx]: "Back" wrapped

Comment (etuzimek) [reply src/components/Cart/Cart.tsx]: yes, it is

File: src/components/Cart/Cart.tsx (hunk 2 above)
Comment (etuzimek) [reply src/components/Cart/Cart.tsx:25]: color changed

**Review by weswojciech — APPROVED** [review]

**Pull request merged**